etc/ci/upload_nightly.sh

```diff
@@ -13,6 +13,7 @@ usage() {
 
 upload() {
     s3cmd put "${2}" "s3://servo-builds/nightly/${1}/"
+    s3cmd cp "s3://servo-builds/nightly/${1}/${2}" "s3://servo-builds/nightly/${1}/servo-latest.${3}"
```

Comment:
This will not work properly - we only need the filename part, not the full path of "${2}". Also, it would be nice to use a local readonly variable to hold the `s3://servo-builds/nightly/` prefix.